pkg/task/obcluster_flow.go

```diff
@@ -84,7 +85,10 @@ func UpgradeOBCluster() *TaskFlow {
 		OperationContext: &v1alpha1.OperationContext{
 			Name:         flowname.UpgradeOBCluster,
 			Tasks:        []string{taskname.ValidateUpgradeInfo, taskname.BackupEssentialParameters, taskname.UpgradeCheck, taskname.BeginUpgrade, taskname.RollingUpgradeByZone, taskname.FinishUpgrade, taskname.RestoreEssentialParameters},
-			TargetStatus: clusterstatus.Running,
+			TargetStatus: clusterstatus.Running, // 能不能升级（兼容性问题，升级兼容性文件， 2. 改一些参数，额外运行一些脚本，用于还原参数  3. 升级的时候做一个检查，升级前检查，检查是否可以升级 4.执行升级脚本，包括会改一些内部表 5.替换镜像，启动一个新的容器，需要一个zone一个zone做,要满足多数派 6. 跑一个脚本 7.把参数设置回去），
```

Comment:
comment in english

Reply:
This was a useless comment and has been removed.